src/site/content/en/learn/css/sizing/index.md

```diff
@@ -0,0 +1,435 @@
+        like resizing a header font based on the width of the page so as the user resizes,
+        the font will also resize.</td>
+      </tr>
+      <tr>
+        <td><a href="https://www.w3.org/TR/css-values-4/#vh">vh</a></td>
+        <td>1% of viewport's height.
+        You can use this to arrange items in a UI,
+        if you have a footer toolbar for example.</td>
+      </tr>
+      <tr>
+        <td><a href="https://www.w3.org/TR/css-values-4/#vi">vi</a></td>
+        <td>1% of viewport's size in the root element's <a href="https://www.w3.org/TR/css-writing-modes-4/#inline-axis">inline axis</a>.
+        Axis refers to writing modes.
+        In horizontal writing modes like English,
+        the inline axis is horizontal.
+        In vertical writing modes like some Japanese typefaces, the inline axis runs top to bottom.</td>
+      </tr>
+      <tr>
+        <td><a href="https://www.w3.org/TR/css-values-4/#vb">vb</a></td>
+        <td>1% of viewport's size in the root element's <a href="https://www.w3.org/TR/css-writing-modes-4/#block-axis">block axis</a>.
+        For the block axis, this would be the directionality of the language.
+        LTR languages like English would have a vertical block axis,
+        since English language readers parse the page from top to bottom.
+        A vertical writing mode has a horizontal block axis.</td>
+      </tr>
+      <tr>
+        <td><a href="https://www.w3.org/TR/css-values-4/#vmin">vmin</a></td>
+        <td>1% of the viewport's smaller dimension.</td>
+      </tr>
+      <tr>
+        <td><a href="https://www.w3.org/TR/css-values-4/#vmax">vmax</a></td>
+        <td>1% of the viewport's larger dimension.</td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+
+```css
+div {
+	width: 10vw;
+}
+
+p {
+	max-width: 60ch;
+}
+```
+<figure class="w-figure">
+{% Codepen {
+  user: 'web-dot-dev',
+  id: 'JjEYqXa'
+} %}
+</figure>
+
+In this example, the `div` will be 10% of the viewport's width because `1vw` is **1% of the viewport width**.
+The `p` element has a `max-width` of `60ch`
+which means it can't exceed the width of 60 "0" characters in the calculated font and size.
+
+{% Aside 'objective' %}
+
+By sizing text with relative units like `em` or `rem`,
+rather than an absolute unit, like `px`, the size of your text can respond to user preferences.
+This can include the system font size or parent element's font size, such as the `<body>`.
+The base size of the `em` is the element's parent and the base size of the `rem` is the base font size of the document.
+
+If you don't define a `font-size` on your `html` element,
+this user-preferred system font size will be honoured if you use relative lengths,
+such as `em` and `rem`.
+If you use `px` units for sizing text,
+this preference will be ignored.
+
+{% Video src="video/VbAJIREinuYvovrBzzvEyZOpw5w1/cUEl77VN4ZtAdElV4fd6.mp4" %}
+{% endAside %}
+
+## Miscellaneous units
```

Comment:
maybe add some subheaders for these misc units? it feels like it trails off and thins out, and could be fixed with some clear headers?

Reply:
Agreed, I've just pushed a little change to that section. @andy-piccalilli maybe take a quick look?